pkgs/os-specific/linux/tp_smapi/default.nix

```
@@ -2,37 +2,20 @@
   stdenv,
   lib,
   fetchFromGitHub,
-  fetchpatch,
   kernel,
-  writeScript,
-  coreutils,
-  gnugrep,
-  jq,
-  curl,
-  common-updater-scripts,
-  runtimeShell,
 }:
 
 stdenv.mkDerivation rec {
   name = "tp_smapi-${version}-${kernel.version}";
-  version = "0.43";
+  version = "0-unstable-2025-05-26";
 
   src = fetchFromGitHub {
     owner = "linux-thinkpad";
     repo = "tp_smapi";
-    rev = "tp-smapi/${version}";
-    sha256 = "1rjb0njckczc2mj05cagvj0lkyvmyk6bw7wkiinv81lw8m90g77g";
+    rev = "a6122c0840c36bf232250afd1da30aaedaf24910";
+    sha256 = "sha256-4bVyhTVj29ni9hduN20+VEl5/N0BAoMNMBw+k4yl8Y0=";
```

```suggestion
    hash = "sha256-4bVyhTVj29ni9hduN20+VEl5/N0BAoMNMBw+k4yl8Y0=";
```
Nit while we're updating this line